examples/src/main/java/com/mapbox/navigation/examples/internal/MapboxRouteLineActivity.java

```diff
@@ -138,28 +138,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         )
         .build();
 
-    TileStore tileStore =
-        TileStore.getInstance(path);
-    ResourceOptions resourceOptions = new ResourceOptions.Builder()
-        .accessToken(getMapboxAccessTokenFromResources())
-        .assetPath(getFilesDir().getAbsolutePath())
-        .cachePath(getFilesDir().getAbsolutePath() + "/mbx.db")
-        .cacheSize(100_000_000L) // 100 MB
-        .tileStore(tileStore)
-        .build();
-    MapInitOptions mapboxMapOptions = new MapInitOptions(
-        this,
-        resourceOptions,
-        MapInitOptions.Companion.getDefaultMapOptions(this),
-        null,
-        false,
-        null
-    );
-    mapboxMapOptions.setResourceOptions(resourceOptions);
-    mapView = new MapView(this, mapboxMapOptions);
-    RelativeLayout mapLayout = findViewById(R.id.mapView_container);
-    mapLayout.addView(mapView);
-
+    mapView = findViewById(R.id.mapView);
```

Comment:
This removes the predictive caching setup from this test activity, and it was the only one that had it.

It was buried here in the first place, so I don't mind dropping it as long as we can build a new one. @korshaknn @Guardiola31337 could you help with that? We'd need some simple test activity that integrates predictive caching.

Reply:
Could you cut a ticket so we don't forget? 🙏